scrapbook-tutorial/src/Comment.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+
+function CommentBody(props) {
+  const rows = props.characterData.map((row, index) => {
+    return (
+      <div className="comment" key={index}>
+        <ul class="commentwrapper">
+          <li class="comment-name">
+            <label>{row.name}</label>
+          </li>
+          <li class="comment-mesage">
+            <label>{row.comment}</label>
+          </li>
+        </ul>
+      </div>
+    );
+  });
+
+  return <div className="comments">{rows}</div>;
+}
+
+function Comment(props) {
```

Comment:
I don't think this does anything.  You could export default the CommentBody function and I think it would render the same way.  If you rewrote line 3 to be: `export default ((props) => {` You could get rid of everything after line 21.

Reply:
Nice one

Reply:
Done